src/Compilers/CSharp/Test/Semantic/Semantics/GenericConstraintsTests.cs

```diff
@@ -1827,9 +1827,9 @@ public void UnmanagedConstraint_Compilation_ReferenceType()
             var c = CreateCompilation("public class Test<T> where T : class, unmanaged {}");
 
             c.VerifyDiagnostics(
-                // (1,39): error CS8380: The 'unmanaged' constraint must come before any other constraints
+                // (1,39): error CS8869: The 'unmanaged' constraint cannot be combined with the 'class' constraint
```

Comment:
>error CS8869: The 'unmanaged' constraint cannot be combined with the 'class' constraint [](start = 27, length = 87)

It looks like the error number and the wording no longer matches the latest revision. Consider updating. in other tests too.